templates/semantic-ui/invenio_app_rdm/frontpage.html

```diff
@@ -22,46 +22,41 @@
   {%- endblock intro_section %}
 
   {%- block grid_section %}
-    <div class="ui container overlapping">
+    <div class="ui container rel-mt-2" id="focus-on">
       <div class="ui stackable grid center-tablet-mobile">
-        <div class="column eight wide transparent-background pl-0 pr-0 rel-pb-3">
-          <div class="highlight-background communities-box info-box rel-pb-3">
-            <div class="ui grid middle aligned">
-              <div class="sixteen wide column"><h2 class="ui header">Communities</h2>
-              </div>
-              <div class="sixteen wide mobile ten wide computer sixteen wide tablet column">
-                <p> Browse the catalogue <br/> curated by CDS users and communities</p>
-              </div>
-              <div class="sixteen wide mobile six wide computer sixteen wide tablet column centered">
-                <i class="huge grey users icon mb-20"></i><br/>
-                <a href="/communities" class="ui secondary button">See all communities</a>
-              </div>
-            </div>
-
-          </div>
-        </div>
 
-        <div class="column eight wide inversed-background info-box">
+        <div class="column nine wide ">
           <div class="ui grid relaxed middle aligned">
-            <div class="sixteen wide column">
-              <h2 class="ui header rel-mb-2">Focus on</h2>
+            <div class="three wide column">
+              <h3 class="ui header">Focus on</h3>
```

Comment:
```suggestion
              <h2 class="ui medium header">Focus on</h3>
```

Reply:
why medium ? I tried to have it in h3 size

Reply:
Yes, h3 is medium :) 
![Screenshot 2023-05-24 at 11 09 12](https://github.com/CERNDocumentServer/cds-rdm/assets/21052053/88c72125-113a-4850-a44a-2e54c7aa2092)
![Screenshot 2023-05-24 at 11 09 15](https://github.com/CERNDocumentServer/cds-rdm/assets/21052053/99bf7c24-3ef0-4ba0-ac89-820a1a347239)